checkIn/templates/admin/warnings.html

```diff
@@ -0,0 +1,75 @@
+{% extends "layout.html" %}
+{% block data %}
+<div class="container">
+	{% if error %}
+        <div class="row">
+            <div class="alert alert-danger text-center">
+                <strong>{{ error }}</strong>
+            </div>
+        </div>
+    {% endif %}
+	<div class="row">
+	    <div class="panel panel-default">
+	        <div class="panel-heading">
+	            <a href="/admin/lookup?sid={{ warnee.sid }}" class="btn btn-primary"><i class="glyphicon glyphicon-chevron-left"></i></a>
+	        </div>
+	        <ul class="list-group">
+	            <li class="list-group-item">
+		            <h3>Warn {{ warnee.name }}</h3>
+		            <form action="/admin/warnings/{{ warnee.sid }}" method="post">
+						<label for="reasonInput">Reason</label>
```

Comment:
For the form here you should take a look at [Bootstrap's form documentation](https://getbootstrap.com/docs/3.4/css/#forms-horizontal) and wrap everything in `div.form-group`s to fix the spacing on the page.